opendebates/models.py

```diff
@@ -41,6 +41,11 @@ class SiteMode(CachingMixin, models.Model):
         help_text="Enter time that debate starts in timezone %s" % settings.TIME_ZONE,
     )
     debate_state = models.CharField(max_length=5, null=True, blank=True)
+    previous_debate_time = models.DateTimeField(
+        null=True, blank=True,
+        help_text=("Enter time that the previous debate occurred in timezone %s to"
+                   " reset current vote counts" % settings.TIME_ZONE)
```

Comment:
"to reset current vote counts" is a bit misleading if the admin action remains separate - maybe just "to enable Votes Since Previous Debate sort option"?